### PR TITLE
Feat: add StatCard component

### LINKED ISSUE #67: Add StatCard component
## Description
Add the StatCard component:
https://www.figma.com/file/nPJGejQqgLQfkVMJqhcCeO/Maquettes?node-id=137%3A1317

It should also be integrated into the ResultDisplay page in this PR.

### PR DESCRIPTION
**Pull request by c1-ra**
## Linked issues
Closes #67

### CODE REVIEW

File: src/views/UI/index.tsx [hunk 1]
```diff
@@ -389,6 +395,63 @@ export const UI: FC = () => {
           <Tooltip icon={AlertCircle} className="mr-3" text="Tooltip text" />
         </div>
       </div>
+      <div className="mb-4">
+        <Typography className="mb-4" element="h1">
+          Results
+        </Typography>
+        <div className="f f-direction-row f-ai-end mb-3">
```
Comment (Damien-MATHIEU) [inline src/views/UI/index.tsx:402]: Very nice!
I would suggest to delete the `align-items: flex-end;`, it makes that every result will be the same height on the same line. I think it is possible to achieve the same thing doing something cleaner, but i do not have it on the top of my head right now

```suggestion
        <div className="f f-direction-row mb-3">
```

**Comment by Damien-MATHIEU** [comment]
And in a second time when we will have more time, i would suggest to use `display: grid;` on the div containing the 3 cards instead of `display: flex;`. 
This way we will have more control on the result cards.
Of course we will talk about it all together :)

**Comment by Damien-MATHIEU** [comment]
But beside the first comment on the height of the cards, the PR is ready to merge in my opinion.

**Comment by c1-ra** [comment]
@Damien-MATHIEU Concerning `display: grid;`, I'll keep it in mind, but is it ok to go ahead with `flex` for the moment.
After discussing it with @GregoryAlbouy and @moreirathomas, we have also decided to move this Component and use this directly.

**Comment by moreirathomas** [comment]
Nice 🤩 

Nitpick : `statCard.module.scss` -> `stat-card.module.scss`.
Then we can merge !

**Review by moreirathomas — APPROVED** [review]

**Pull request merged**